moveit_ros/planning/robot_model_loader/src/robot_model_loader.cpp

```diff
@@ -166,15 +175,21 @@ void RobotModelLoader::configure(const Options& opt)
           if (has_position_limits && canSpecifyPosition(joint_model, joint_id))
           {
             const auto max_pos_name = prefix + "max_position";
-            if (!node_->has_parameter(max_pos_name))
+            try
+            {
+              node_->declare_parameter(param_name);
```

Comment:
Here you declare `param_name` while ignoring `max_pos_name` from above. Same for `min_pos_name`